server/preprocessing/other-scripts/test/test-openaire.R

```diff
@@ -0,0 +1,34 @@
+rm(list = ls())
+
+library(rstudioapi)
+
+options(warn=1)
+
+wd <- dirname(rstudioapi::getActiveDocumentContext()$path)
+
+setwd(wd) #Don't forget to set your working directory
+
+query <- "247153" #args[2]
```

Comment:
As discussed, query will be the acronym of the project. The relevant data for retrieving the project (funding stream & grant id) will be sent as parameters `funding_stream` and `grant_id`.